api/src/index.js

```diff
@@ -1,72 +1,71 @@
 #!/usr/bin/env node
-require('nocamel');
-const Logger = require('logplease');
-const express = require('express');
-const expressWs = require('express-ws');
-const globals = require('./globals');
-const config = require('./config');
-const path = require('path');
-const fs = require('fs/promises');
-const fss = require('fs');
-const body_parser = require('body-parser');
-const runtime = require('./runtime');
-
-const logger = Logger.create('index');
+import { create, setLogLevel } from 'logplease';
+import express from 'express';
+import expressWs from 'express-ws';
+import * as globals from './globals';
+import config from './config';
+import { join } from 'path';
+import { readdir } from 'fs/promises';
+import { existsSync, mkdirSync, chmodSync } from 'fs';
+import { urlencoded, json } from 'body-parser';
+import { load_package } from './runtime';
+
+const logger = create('index', {});
 const app = express();
 expressWs(app);
 
 (async () => {
-    logger.info('Setting loglevel to', config.log_level);
-    Logger.setLogLevel(config.log_level);
+    logger.info('Setting loglevel to');
+    // @ts-ignore
+    setLogLevel(config.log_level);
```

Comment:
`config.log_level` is typed `string`
`setLogLevel` is typed `"INFO" | "DEBUG" | "WARN" | "ERROR" | "NONE"`